packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -31,6 +31,7 @@ dayjs.extend(timeZone)
 dayjs.extend(advancedFormat)
 
 Sentry.init({
+  environment: process.env.NODE_ENV !== 'production' ? 'testing' : 'production',
```

Comment:
Can we just do
```suggestion
  environment: process.env.NODE_ENV,
```

Reply:
Updated, the possible values for the environment are now - "development" | "production" | "test"